server/plugin_test.go

```diff
@@ -95,3 +96,216 @@ func TestExecuteCommand(t *testing.T) {
+				return "https://test.com"
+			})
+
+			monkey.PatchInstanceMethod(reflect.TypeOf(&Plugin{}), "GetPluginURL", func(_ *Plugin) string {
+				return "https://test.com/api/v4/actions/dialogs/open"
+			})
+
+			roles := "system_user"
+			if val.isAdmin {
+				roles += " system_admin"
+			}
+
+			mockAPI.On("GetUser", mock.AnythingOfType("string")).Return(&model.User{Id: "123", Roles: roles}, nil)
+
+			res, err := p.ExecuteCommand(&plugin.Context{}, val.commandArgs)
+			assert.Nil(t, err)
+			assert.NotNil(t, res)
+		})
+	}
+}
+
+func TestConfigAddCommand(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	p.API = mockAPI
+	for name, val := range map[string]struct {
+		commandArgs      *model.CommandArgs
+		ephemeralMessage string
+		isAdmin          bool
+	}{
+		"admin restriction on config add command": {
+			commandArgs:      &model.CommandArgs{Command: "/confluence config \"add\"", UserId: userID, ChannelId: channelID},
+			ephemeralMessage: installOnlySystemAdmin,
+			isAdmin:          false,
+		},
+		"config add command": {
+			commandArgs: &model.CommandArgs{Command: "/confluence config \"add\"", UserId: userID, ChannelId: channelID},
+			isAdmin:     true,
+		},
+	} {
+		t.Run(name, func(t *testing.T) {
+			defer monkey.UnpatchAll()
+			mockAPI := baseMock()
+
+			mockAPI.On("SendEphemeralPost", mock.AnythingOfType("string"), mock.AnythingOfType("*model.Post")).Run(func(args mock.Arguments) {
+				post := args.Get(1).(*model.Post)
+				if val.ephemeralMessage != "" {
+					assert.Equal(t, val.ephemeralMessage, post.Message)
+				}
+			}).Once().Return(&model.Post{})
+
+			roles := "system_user"
+			if val.isAdmin {
+				roles += " system_admin"
+			}
+
+			mockAPI.On("GetUser", mock.AnythingOfType("string")).Return(&model.User{Id: "123", Roles: roles}, nil)
+
+			res, err := p.ExecuteCommand(&plugin.Context{}, val.commandArgs)
+			assert.Nil(t, err)
+			assert.NotNil(t, res)
+		})
+	}
+}
+
+func TestConfigListCommands(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	p.API = mockAPI
+	p.instanceStore = p.getMockInstanceStoreKV(1)
+	for name, val := range map[string]struct {
+		commandArgs      *model.CommandArgs
+		ephemeralMessage string
+		isAdmin          bool
+		patchAPICalls    func()
+	}{
+		"admin restriction on config list command": {
+			commandArgs:      &model.CommandArgs{Command: "/confluence config ", UserId: userID, ChannelId: channelID},
+			ephemeralMessage: installOnlySystemAdmin,
+			isAdmin:          false,
+		},
+		"config list command no config saved": {
+			commandArgs:      &model.CommandArgs{Command: "/confluence config \"list\"", UserId: userID, ChannelId: channelID},
+			ephemeralMessage: noSavedConfig,
+			isAdmin:          true,
+			patchAPICalls: func() {
+				monkey.PatchInstanceMethod(reflect.TypeOf(&Plugin{}), "GetConfigKeyList", func(_ *Plugin) ([]string, error) {
+					return []string{}, nil
+				})
+			},
+		},
+		"config list command": {
```

Comment:
```suggestion
		"config list command success": {
```
Fix this everywhere.